services/movie_session.py

```diff
@@ -0,0 +1,51 @@
+from django.db.models import QuerySet
+
+from db.models import MovieSession
+
+
+def create_movie_session(
+        movie_show_time: int,
```

Comment:
The `movie_show_time` parameter should be a `datetime` object instead of an `int` to accurately represent the show time.